jwql/website/apps/jwql/templates/archive.html

```diff
@@ -2,73 +2,118 @@
-	<!-- <link href="{{staticURL}}css/loader.css?{% now 'U' %}" rel="stylesheet"> -->
 
 {% endblock %}
 
 {% block content %}
 
     <main role="main" class="container">
 
-    	<h3 id='title'>Archived {{ inst }} Images</h3>
-
-    	<p id='img_show_count'>Showing {{ num_proposals }}/{{ num_proposals }} proposals
-    		<a href="https://jwst-docs.stsci.edu/display/JDAT/File+Naming+Conventions+and+Data+Products" target="_blank" style="color: black">
-    			<span class="help-tip mx-2">i</span>
-    		</a>
-    	</p>
-
-    	<hr>
-
-    	<!-- Filter and sort options -->
-		<div class="d-flex flex-row">
-			<!-- Search Bar -->
-		    <div class="input-group mb-3">
-			  <input id='search_box' type="text" class="form-control" placeholder="Search" aria-label="Search by proposal" oninput="search('{{ num_proposals }}');">
-			  <div class="input-group-append">
-			  	<span class="input-group-text"><span class="fas fa-search" aria-hidden="true"></span></span>
-			    <!--<button class="btn btn-primary" type="button"><span class="fas fa-search" aria-hidden="true"></span></button>-->
-
-			  </div>
-			</div>
-
-			<!-- Sort Options -->
-			<div class="mx-4 mr-5 ml-auto">
-		    	Sort by:
-		    	<span class="dropdown">
-				  <button class="btn btn-primary dropdown-toggle ml-3" type="button" id="sort_dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Ascending</button>
-				  <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
-				  	<a class="dropdown-item" href="#" onclick="sort_by_proposals('Ascending');">Ascending</a>
-			    	<a class="dropdown-item" href="#" onclick="sort_by_proposals('Descending');">Descending</a>
-				  </div>
-				</span>
-			</div>
-		</div>
-
-		<hr>
-
-		<!-- Display the data -->
-		<div id='proposal-array'>
-		{% for prop, thumb, n in zipped_thumbnails %}
-			<div class="proposal text-center">
-				<a href={{ url('jwql:archive_thumb', args=[inst, prop]) }} id="proposal{{ loop.index }}" proposal="{{prop}}">
-					<span class="helper"></span><img src='{{ static("thumbnails/") }}{{thumb}}' alt='' width=100%>
-					<div class='proposal-color-fill' ></div>
-					<div class='proposal-info'>
-						<h3>{{prop}}</h3>
-						<h6>{{n}} Files</h6>
-					</div>
-				</a>
-				<!-- <a -->
-			</div>
-		{% endfor %}
-		</div>
-		<a id="no_proposals_msg" style='display: none'>No proposals match the search criteria.</a>
-
-	</main>
+        <h3 id='title'>Archived {{ inst }} Images</h3>
+
+        <p id='img_show_count'></p>
+        <hr>
+
+        <!-- Filter and sort options -->
+        <div class="d-flex flex-row">
+            <!-- Search Bar -->
+            <div class="input-group mb-3">
+              <input id='search_box' type="text" class="form-control" placeholder="Search" aria-label="Search by proposal" oninput="search('{{ num_proposals }}');">
```

Comment:
So close!! Unfortunately, now I am seeing 
> Showing 17/{{ num_proposals }} proposals

when I type something into that search bar in `archive.html`. I think that it's because of how this `num_proposals` variable is getting passed into the javascript `search()` funtion.

Problem is, this looks completely correct to me. Do you have any ideas why that might be happening?

Reply:
@laurenmarietta Ahh good catch again! :scream: It is indeed because of the way `num_proposals` is being passed.  The problem is that the way I restructured things, the variable is no longer available in the view.  I'll have to come up with a workaround.